packages/nocodb-sdk/src/lib/columnHelper/columns/Links.ts

```
@@ -0,0 +1,68 @@
+    let parsedVal = value;
+
+    try {
+      parsedVal = typeof value === 'string' ? JSON.parse(value) : value;
+    } catch {}
+
+    if (
+      !(
+        parsedVal &&
+        ncIsObject(parsedVal) &&
+        ['rowId', 'columnId', 'fk_related_model_id', 'value'].every((key) =>
+          (parsedVal as Object).hasOwnProperty(key)
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)